src/containers/Review/ReviewPage.tsx

```diff
@@ -67,10 +74,28 @@ const ReviewPage: React.FC<{
       current: { stage },
       name,
     },
+    thisReview,
     attemptSubmission,
     firstIncompleteReviewPage,
   } = fullReviewStructure
 
+  const status = thisReview?.status || null
+
+  if (status === ReviewStatus.Pending && showWarningModal.open === false) {
+    const { title, message, option } = messages.REVIEW_STATUS_PENDING
+    setShowWarningModal({
+      open: true,
+      title,
+      message,
+      option,
+      onClick: () => {
+        setShowWarningModal({ open: false })
+        push(`/application/${fullReviewStructure.info.serial}/review`)
+      },
+      onClose: () => setShowWarningModal({ open: false }),
```

Comment:
do we want to make sure we navigate back to home either way (even is modal is somehow closed ?)

Reply:
Yea. Good point

Reply:
Done: da72742